### DISCUSSION TITLE
how to access to internet from security onion for send alerts by grafana to telegram or slack ?

### DISCUSSION THREAD

**Discussion by mJAVAdzk** [Security-Onion-Solutions/securityonion#6384] [Unsupported Versions]
lvl=eror msg="Failed to send slack notification" logger=alerting.notifier.slack error="Post \"https://hooks.slack.com/services/XXXXXXXXXXXXXXXXXXXXXXXXXXXX": dial tcp 34.193.255.5:443: i/o timeout (Client.Timeout exceeded while awaiting headers)" webhook="Grafana slack channel"

**Comment by dougburks** [comment]
When you installed Security Onion, did you give it a proper IP address and gateway so that it could reach the Internet?

What is the output of `nc -vz hooks.slack.com 443`?
